valter-junior/projetoIP
# feature: As imagens e os sons usados

## What I did

The pictures and sound effects in the game.

## How to test

1. Open the terminal and go to the folder of this project

2. In terminal:
   1. `git checkout main`
   1. `git pull`
   1. `git checkout feature\Imagens`
   1. `git pull origin feature\Imagens`

3. Test the things that you think are worth testing, even the ones that are not in this description

test if the is working how expected;
test if it's working on the main;
